src/sentry/monitors/serializers.py

```diff
@@ -128,13 +129,45 @@ class MonitorSerializerResponse(TypedDict):
 
 @register(MonitorCheckIn)
 class MonitorCheckInSerializer(Serializer):
+    def __init__(self, start=None, end=None, expand=None, organization_id=None, project_id=None):
+        self.start = start
+        self.end = end
+        self.expand = expand
+        self.organization_id = organization_id
+        self.project_id = project_id
```

Comment:
The `start` and `end` are pretty ambiguous here, can we document these somehow, what are they used for?